react-components/src/components/NodeCacheProvider/AnnotationModelUtils.ts

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type CogniteClient, type Asset } from '@cognite/sdk/dist/src';
+import { uniqBy, chunk } from 'lodash';
+import { filterUndefined } from '../../utilities/filterUndefined';
+import { type RevealAnnotationModel } from './types';
+import { getAssetIdOrExternalIdFromAnnotation } from './utils';
+
+export async function fetchAnnotationAssets(
+  annotations: RevealAnnotationModel[],
+  sdk: CogniteClient
+): Promise<Map<number, Asset>> {
```

Comment:
Would be nice if there was a type alias to explain what the `number` key is in this Map. (Is it annotation ID or asset ID?)

Reply:
Are you referring like `CogniteInternalId` for `assetId` which is basically `number`?